src/scenes/GameScene.js

```diff
@@ -166,7 +169,8 @@ export default class Game extends Phaser.Scene {
           motion: "idle",
           name: player.name,
           scene: this,
-          ...this.groundLayer.tileToWorldXY(player.x, player.y),
+          // we need to add 1 to X to render player in the right place
+          ...this.groundLayer.tileToWorldXY(player.x + 1, player.y),
```

Comment:
That's second time where this happens, should be probably wrapped into a separate function.